bref/symfony-bridge/0.1/serverless.yaml

```
@@ -0,0 +1,45 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: symfony
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: dev
+    runtime: provided.al2
+    environment:
+        # Symfony environment variables
+        APP_ENV: prod
+
+plugins:
+    - ./vendor/bref/bref
+
+functions:
+    # This function runs the Symfony website/API
+    web:
+        handler: public/index.php
+        timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
+        layers:
+            - ${bref:layer.php-80-fpm}
```

Based on
https://github.com/brefphp/symfony-bridge/blob/6156395329df3bb34a9a17c5e440ca8ab76c173f/composer.json#L22

you can use the bridge with PHP 7.3, 7.4, but everything in the recipe contains php 8 🤔

---

Can't see the problem with that :thinking: 

IMHO We're just providing a decent default (7.3 is EOL and 8.0 is now supported by a fair amount of packages)

If they want, users can replace this line with a previous version this way and it will work

```suggestion
            - ${bref:layer.php-74-fpm}
```